api/internal/humiographql/filter-alerts.go

```diff
@@ -0,0 +1,49 @@
+package humiographql
+
+import graphql "github.com/cli/shurcooL-graphql"
+
+type FilterAlert struct {
+	ID             graphql.String   `graphql:"id"`
+	Name           graphql.String   `graphql:"name"`
+	Description    graphql.String   `graphql:"description"`
+	QueryString    graphql.String   `graphql:"queryString"`
+	Actions        []Action         `graphql:"actions"`
+	Labels         []graphql.String `graphql:"labels"`
+	Enabled        graphql.Boolean  `graphql:"enabled"`
+	LastTriggered  Long             `graphql:"lastTriggered"`
+	LastErrorTime  Long             `graphql:"lastErrorTime"`
+	LastError      graphql.String   `graphql:"lastError"`
+	LastWarnings   []graphql.String `graphql:"lastWarnings"`
+	QueryOwnership QueryOwnership   `graphql:"queryOwnership"`
+}
+
+type Action struct {
+	Name        graphql.String `graphql:"name"`
+	DisplayName graphql.String `graphql:"displayName"`
+	ID          graphql.String `graphql:"id"`
```

Comment:
Do we use the id for anything?

Reply:
No, we did earlier in the development stage.

Reply:
Removed